modules/nw-udn-additional-config-details.adoc

```diff
@@ -20,9 +20,22 @@ The following table explains additional configurations for UDN that are optional
 
 The `joinSubnets` field configures the routing between different segments within a user-defined network. Dual-stack clusters can set 2 subnets, one for each IP family; otherwise, only 1 subnet is allowed. This field is only allowed for the `Primary` network.
 
-|`spec.IPAMLifecycle`
+|`spec.ipam.lifecycle`
 |object
-|The `IPAMLifecycle` field configures the IP address management system (IPAM). You might use this field for virtual workloads to ensure persistent IP addresses. This field is allowed when `topology` is `layer2`. The `subnets` field must be specified when this field is specified. Setting a value of `Persistent` ensures that your virtual workloads have persistent IP addresses across reboots and migration. These are assigned by the container network interface (CNI) and used by OVN-Kubernetes to program pod IP addresses. You must not change this for pod annotations.
+|The `spec.ipam.lifecycle` field configures the IP address management system (IPAM). You might use this field for virtual workloads to ensure persistent IP addresses. The only allowed value is `Persistent`, which 
+ensures that your virtual workloads have persistent IP addresses across reboots and migration. These are assigned by the container network interface (CNI) and used by OVN-Kubernetes to program pod IP addresses. You must not change this for pod annotations. 
+
+Setting a value of `Persistent` is only supported when `spec.ipam.mode` is set to `Enabled`. When using the `spec.ipam.lifecycle` field, you must either populate the `subnets` field, or set `spec.ipam.mode` to `disabled`.
+
+|`spec.ipam.mode`
+|object
+|Controls how much of the IP configuration is managed by OVN-Kubernetes. The following options are available:
+
+**Enabled:** +
```

Comment:
here you could say somewhere that subnets must be empty when ipam.mode is Disabled and must be not-empty when it is Enabled

Reply:
Addressed